src/tools/miri/tests/utils/mod.rs

```diff
@@ -1,4 +1,5 @@
 #![allow(dead_code)]
+#![allow(unused_imports)]
```

Comment:
In this case, it's under the ﻿tests directory and none of the re-exports have been used, so it will be detected. Therefore, adding `﻿#![allow(unused_imports)]` is necessary.

By the way, once this PR is merged, it will throw `﻿unused_imports` warnings for re-export statements that are solely intended to re-export something in test files. Could this potentially cause confusion for users?

Reply:
Well, if some code is only used for tests it is better to put it under `#[cfg(test)]`.
Preferably by moving into `mod tests` files, if possible.